### ISSUE TITLE
[BUG] Surface map_to_image assertion failed

### ISSUE THREAD

**Issue by danielronnkvist** [gtk-rs/gtk-rs-core#265]
**Bug description**

Trying to use the `map_to_image` method on a ImageSurface but I'm getting an error:
>Assertion failed: (CAIRO_REFERENCE_COUNT_HAS_REFERENCE (&surface->ref_count)), function cairo_surface_reference, file cairo-surface.c, line 930.

Even with a minimal example like this it fails:
```rust
fn main() {
    let surface = cairo::ImageSurface::create(cairo::Format::ARgb32, 1080, 1080).unwrap();
    let mapped = surface.map_to_image(Some(cairo::RectangleInt {
        x: 100,
        y: 100,
        width: 500,
        height: 500
    })).unwrap();
}
```

**Comment by sdroege** [comment]
Thanks for reporting. This should be fixed by https://github.com/gtk-rs/gtk-rs-core/pull/266 but let me know if there's more broken around there.

**Comment by danielronnkvist** [comment]
Thank you @sdroege!

**Comment by sdroege** [comment]
I'll create a bugfix release for crates.io once you confirmed that there's not more broken, some time in the next days :)

**Comment by danielronnkvist** [comment]
@sdroege It works as intended for my use case, thanks!

**Comment by sdroege** [comment]
Thanks for confirming!

**Comment by sdroege** [comment]
Released as 0.14.7 on crates.io now
